test/e2e-tests-latest-eventing.sh

```diff
@@ -28,7 +28,10 @@
 
 source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/e2e-tests.sh
 
-OPERATOR_DIR=$(dirname $0)/..
+# inherited from vendor/knative.dev/test-infra/scripts/library.sh
+declare -A IS_PROW
+
+OPERATOR_DIR=$(cd $(dirname "$0")/.. && pwd)
```

Comment:
Changed 
`OPERATOR_DIR=$(dirname $0)/..` to
`OPERATOR_DIR=$(cd $(dirname "$0")/.. && pwd)`
. 

First one results in a relative path and when we `cd` into `eventing` folder later in the script, `cd`ing back with that relative path goes to another directory relative to `eventing`, not `eventing-operator`.

Second one results in an absolute path and `cd`ing to that path always works.